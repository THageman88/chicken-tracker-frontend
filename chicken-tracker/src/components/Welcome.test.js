import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter } from 'react-router-dom';
import WelcomePage from "./Welcome";

it("renders without crashing", function() {
  render(
    <BrowserRouter>
      <WelcomePage />
    </BrowserRouter>
  );
});

it("matches snapshot", function(){
  const {asFragment}= render(<BrowserRouter>
    <WelcomePage />
  </BrowserRouter>);
  expect (asFragment()).toMatchSnapshot();
});

