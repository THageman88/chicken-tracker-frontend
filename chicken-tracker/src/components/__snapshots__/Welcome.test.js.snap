// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1
      class="text-white"
    >
       Welcome to Chicken Tracker!
    </h1>
    <div>
      <p
        class="text-white"
      >
        Log in or Register to continue
      </p>
    </div>
    <div>
      <button
        class="submitBtn"
      >
        <a
          class="submitBtn"
          href="/login"
        >
          Login
        </a>
      </button>
    </div>
    <div>
      <button
        class="submitBtn"
      >
        <a
          class="submitBtn"
          href="/register"
        >
          Register
        </a>
      </button>
    </div>
  </div>
</DocumentFragment>
`;
